src/resources/transfers.ts

```diff
@@ -0,0 +1,52 @@
+import { createRequestMethod } from "../interfaces/server/request-method.interface";
+import ResourceClass from "../classes/abstracts/resource.class";
+import { IndexInterface } from "../interfaces/resources/structures/index.interface";
+
+/**
+ * @endpoint transfers
+ * @url /transfers/
+ */
+class TransfersResource extends ResourceClass {
+	methodDefinitions = {
+		index: this.index(),
+		cancel: this.cancel()
+	};
+
+	constructor() {
+		super("transfers");
+		this.buildAliases();
+	}
+
+	/**
+	 * Cancel a transfer.
+	 * @auth true
+	 * @params {source_or_destination?: uuid, start_utc?: Date, end_utc?: Date}
+	 * @required
+	 * @url /transfers
+	 */
+	index(): IndexInterface {
+		return createRequestMethod({
+			method: "GET",
+			path: "",
+			requiresAuth: true
+		}) as any;
+	}
+
+	/**
+	 * Cancel transfering of a ticket to another user.
+	 * @auth true
+	 * @params {id: uuid}
+	 * @required {id: uuid}
+	 * @url /tickets/transfer
+	 */
+	cancel(): void {
+		return createRequestMethod({
+			method: "DELETE",
+			path: "/{id}",
+			required: ["id"],
```

Comment:
If the field is in the path it is forced to be required
```suggestion
			required: [],
```